### DISCUSSION TITLE
Changing some additional preferences would be perfect

### DISCUSSION THREAD

**Discussion by maxboz** [lardbit/nefarious#189] [Q&A]
Hi! I just installed nefarious but there seems to be no way of changing the language of downloaded subtitles. That wouldn't be a problem if I could tell the app not to change the filenames, because other scripts would find the right subtitles (like jellyfin). But now I'm stuck with English only :P 
Also, being able to tell nefarious to just check for new seasons, without downloading the old ones would be cool too. I'm waiting for some shows to continue, but I've watched previous episodes. 
Thanks for your answers!

**Comment by lardbit** [comment]
> Hi! I just installed nefarious but there seems to be no way of changing the language of downloaded subtitles. That wouldn't be a problem if I could tell the app not to change the filenames, because other scripts would find the right subtitles (like jellyfin). But now I'm stuck with English only :P

Can you clarify a bit more of whats happening?  nefarious [attempts to download subtitles using the chosen language](https://github.com/lardbit/nefarious/blob/master/src/nefarious/opensubtitles.py#L61) (chosen on the settings page) and [names them using that language](https://github.com/lardbit/nefarious/blob/master/src/nefarious/opensubtitles.py#L151-L157).  Include some specific examples of what's happening and I'll be able to troubleshoot easier.


> Also, being able to tell nefarious to just check for new seasons, without downloading the old ones would be cool too. I'm waiting for some shows to continue, but I've watched previous episodes.

I didn't realize this was happening.  It was unintentional if so. I'll have to investigate this one.
